honeybee_vtk/index.py

```diff
@@ -0,0 +1,90 @@
+"""Write index.json to color vtk datasets."""
+
+import os
+import json
+
+# Scarlet colors in decimal
+layer_colors = {
+    'Wall': [0.858, 0.713, 0.560],
+    'Aperture': [0.858, 0.988, 1],
+    'Door': [0.776, 0.847, 0.686],
+    'Shade': [0.937, 0.643, 0.545],
+    'Floor': [0.031, 0.572, 0.647],
+    'RoofCeiling': [0.407, 0.325, 0.411],
+    'Aperture vectors': [1, 1, 1],
+    'grid base': [0.925, 0.250, 0.403],
+    'grid base vectors': [1, 1, 1],
+    'grid mesh' : [0.925, 0.250, 0.403],
+    'grid mesh vectors': [1, 1, 1],
```

Comment:
I think it is better to be consistent and start all the names with Upper case -> Grid mesh vectors, ...